src/lambda/wizards/samDeployWizard.ts

```diff
@@ -592,3 +609,47 @@ async function getTemplateChoices(...workspaceFolders: vscode.Uri[]): Promise<Sa
         return new SamTemplateQuickPickItem(uri, showWorkspaceFolderDetails)
     }).sort((a, b) => a.compareTo(b))
 }
+
+async function populateS3QuickPick(
```

Comment:
Consider adding a function comment (eg: picker is not populated until all bucket listing has completed)